draft-ietf-homenet-front-end-naming-delegation.mkd

```diff
@@ -98,38 +98,55 @@ Infrastructure via a Distribution Master (DM).
 
 #Introduction
 
-IPv6 provides global end to end IP connectivity. End users prefer to use
-names instead of long and complex IPv6 addresses when accessing services
-hosted in the home network.
-
-{{!RFC7368}} recommends that home networks be resilient to connectivity
-disruption from the ISP. The public names should be resolvable within
-the home network and on the Internet, even when there are disruptions.
-This could be achieved by a device inside the home network that builds,
+The Homenet Naming authority is responsible for making devices within
+the home network accessible by name within the home network as well as
+from outside the home network (e.g. the Internet).  IPv6 connectivity
+provides for provides the possibility of global end to end IP connectivity.
```

Comment:
I suspect the sentence could be:
IPv6 provides the possibility of global end to end connectivity.